### DISCUSSION TITLE
Some link in PA#2 that goes to KENS wiki does not match.

### DISCUSSION THREAD

**Discussion by jinnkim0112** [ANLAB-KAIST/KENSv3#144] [Project Q&A 2. Transport Layer (TCP)] [answered]
I have found some mistakes on the google form [PA#2](https://docs.google.com/document/d/1scmD3Ah3GrhBo7FiAZtpCydoFE1eXlDWabNN8QwdTt0/edit#heading=h.b250bkx1wmb3) where some links go to a unintentionally different link and shows no contents in the page. I believe this is due to typos when making the links.

Specifically, [Using Timer](https://github.com/ANLAB-KAIST/KENSv3/wiki/Tip:-Using-a-Timerki/Tip:-Using-Timer) under the subtitle, Close Function
and [Figuring out the source IP address](https://github.com/ANLAB-KAIST/KENSv3/wiki/Tip:-Figuring-out-the-source-IP-address-to-useRetrieving-Route-Information) under the subtitle, A First Step to Build Your Own TCP does not go to the correct page.
Former must go to [here](https://github.com/ANLAB-KAIST/KENSv3/wiki/Tip:-Using-a-Timer) and later must go to [here](https://github.com/ANLAB-KAIST/KENSv3/wiki/Tip:-Figuring-out-the-source-IP-address-to-use).

I have somewhat fixed this issue by making a wiki post to redirect to the correct pages but I hope the TAs fix this issue to not cause confusion.

**Comment by Pusnow** [comment, marked as answer]
Thank you for reporting this issue. I'll fix it soon.

**Comment by faust22** [comment]
you can directly access it via Wiki tab.
